### PR TITLE
Set up Frontend to handle new schemas including sell_map_center

### PR DESCRIPTION
**Pull request by DarinHajou**
The frontend is now ready to handle both the old and new schemas for sellers. Changes where made to the fetchSellerCoordinates function by adding sell_map_center field and improving error handling.

### CODE REVIEW

**Comment by swoocn** [comment]
Thanks @DarinHajou - but, I don't think this PR change is necessary now that the Seller schema has been permanently adjusted to ->
image

All Seller instances should have the adjusted `sell_map_center` object.

File: src/components/shared/map/Map.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import MapMarkerPopup from './MapMarkerPopup';
 // Utility function to ensure coordinates are within valid ranges
 const sanitizeCoordinates = (lat: number, lng: number) => {
   const sanitizedLat = Math.min(Math.max(lat, -90), 90);
-  const sanitizedLng = ((lng + 180) % 360 + 360) % 360 - 180; // Ensures -180 < lng <= 180
+  const sanitizedLng = ((lng + 180) % 360 + 360) % 360 - 180;
```
Comment (swoocn) [inline src/components/shared/map/Map.tsx:15]: 👍

File: src/components/shared/map/Map.tsx [hunk 2]
```diff
@@ -26,17 +26,31 @@ const fetchSellerCoordinates = async (origin: LatLngTuple, radius: number): Prom
...
-      return {
-        ...seller,
-        coordinates: [lat, lng] as LatLngTuple
-      };
-    });
-
-    console.log('Fetched sellers data:', sellersWithCoordinates);
-
-    return sellersWithCoordinates;
+    console.log('Fetched sellers data:', sellersData);
```
Comment (swoocn) [inline src/components/shared/map/Map.tsx:29]: This logic is/ was no longer necessary.

Previously, only those with the unadjusted Seller schema (i.e., **coordinates**) would display as map markers in the live environment, and those with the adjusted Seller schema (i.e., **sell_map_center**) would display as map markers in the development environment.  Now that the development changes have been deployed, the live environment is also synced.  I hope that makes sense.

File: src/components/shared/map/Map.tsx [hunk 3]
```diff
@@ -202,7 +216,7 @@ const Map = ({ center }: { center: LatLngExpression }) => {
         center={origin}
         zoom={13}
         zoomControl={false}
-        className="w-full flex-1 fixed bottom-[0px] h-[calc(100vh-76.19px)] left-0 right-0">
+        className="w-full flex-1 fixed top-[90px] h-[calc(100vh-55px)] left-0 right-0 bottom-0">
```
Comment (swoocn) [inline src/components/shared/map/Map.tsx:219]: 👍

**Comment by swoocn** [comment]
I will approve this PR after you remove the unnecessary **coordinates** vs **sell_map_center** logic.  Thanks @DarinHajou!

**Comment by DarinHajou** [comment]
Hi Danny, Thanks for the review. I'll remove the logic for handling the old coordinates from Map.tsx and resubmit the PR.On Jul 24, 2024 17:18, Danny Lee ***@***.***> wrote:
I will approve this PR after you remove the unnecessary coordinates vs sell_map_center logic.  Thanks @DarinHajou!

**Comment by swoocn** [comment]
This PR is superseded by PR @ [https://github.com/map-of-pi/map-of-pi-frontend-react/pull/132](https://github.com/map-of-pi/map-of-pi-frontend-react/pull/132)

**Pull request closed without merging**